src/styles/global.css

```
@@ -23,6 +23,8 @@ h1 {
 /* nav styles */
 
 .hamburger {
+  background: transparent;

```

Since both the hamburger and the theme switcher are buttons, they are now displayed on a single line (but not aligned) whereas they were previously stacked. So we might want to add the following change to keep the same output:
```suggestion
  display: block;
  background: transparent;
```

---

good catch!